docs-guides/source/convert-tensorflow-2-bert-transformer-models.md

```diff
@@ -12,109 +12,131 @@ The following examples demonstrate converting TensorFlow 2 models to Core ML usi
 
 The following example converts the [DistilBERT model from Huggingface](https://huggingface.co/transformers/model_doc/distilbert.html#tfdistilbertformaskedlm) to Core ML. 
 
-```{admonition} Install Transformers
+```{admonition} Requirements
 
-You may need to first install Transformers version 4.17.0.
+This example requires TensorFlow 2 and Transformers version 4.17.0.
```

Comment:
I need to be specific. It doesn't work with TensorFlow 1. Even though 1 is deprecated, the environment you are using may still have it.

Reply:
Saying "TensorFlow 2" rather than just "TensorFlow" is fine. My main point here is that you shouldn't remove TensorFlow entirely since it looks like the example still uses TensorFlow.